PowerEditor/installer/nativeLang/english.xml

```diff
@@ -13,11 +13,11 @@ Translation note:
 					<Item menuId="file" name="&amp;File"/>
 					<Item menuId="edit" name="&amp;Edit"/>
 					<Item menuId="search" name="&amp;Search"/>
-					<Item menuId="view" name="&amp;View"/>
+					<Item menuId="view" name="&amp;Display"/><!-- Because the term View is also used in the Menu "Move to Other View" and "Clone to Other View" -->
```

Comment:
The term "View" is a convention of Windows applications, and used by the major software under Windows.

Reply:
> The term "View" is a convention of Windows applications, and used by the major software under Windows.

I am well aware of this and if I made this proposal it is just to be able to retain the use of the term view for the right zone and left zone which are called views.

I think that it is in fact this name that a change should be made.
I suggest talking about display area where appropriate.

The lines concerned would be:
328 <Item id="44072" name="Focus on Another View"/>
351 <Item id="10001" name="Move to Other View"/>
352 <Item id="10002" name="Clone to Other View"/>
444 <Item CMDID="10001" name="Move to Other View"/>
445 <Item CMDID="10002" name="Clone to Other View"/>

They could become:
328 <Item id="44072" name="Focus on Another Display Area"/>
351 <Item id="10001" name="Move to Other Display Area"/>
352 <Item id="10002" name="Clone to Other Display Area"/>
444 <Item CMDID="10001" name="Move to Other Display Area"/>
445 <Item CMDID="10002" name="Clone to Other Display Area"/>

This is only a suggestion but it seems to me that it would provide better clarity when using the software.